### PR TITLE
Add new pardef and correct 'semideu'

### PR DESCRIPTION
**Pull request by albertonl**

### CODE REVIEW

File: apertium-cat.cat.dix [hunk 1]
```diff
@@ -5121,6 +5121,13 @@
 
 </pardef>
 
+<pardef n="semide/u__n">
+  <e>       <p><l>eu</l>         <r>u<s n="n"/><s n="m"/><s n="sg"/></r></p></e>
+  <e>       <p><l>essa</l>      <r>u<s n="n"/><s n="f"/><s n="sg"/></r></p></e>
+  <e>       <p><l>us</l>        <r>u<s n="n"/><s n="m"/><s n="pl"/></r></p></e>
+  <e>       <p><l>esses</l>     <r>u<s n="n"/><s n="f"/><s n="pl"/></r></p></e>
+</pardef>
+
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: It's OK to add the feminine form. But we must keep the form with diacritic (semidéu). The words with diacritics changed in the orthography reform will be dealt with in a post-edition module.

**Comment by albertonl** [comment]
I think the word is tricky. For example, in [diccionari.cat](http://www.diccionari.cat/lexicx.jsp?GECART=0123042), 'semidéu' is written with accent. Meanwhile, in other dictionaries, such as the [AVL's dictionary](http://avl.gva.es/lexicval/), or [diccionaris.cat](https://www.diccionaris.cat/?diccionario=77&empieza=empieza&palabra=semid%C3%A9u&imprimir=N) is written without it.
Anyway, I corrected it as you told me. Thanks.

File: apertium-cat.cat.dix [hunk 2]
```diff
@@ -122724,7 +122731,7 @@
 <e lm="rutherfordi" a="albertonl">   <i>rutherfordi</i><par n="abric__n"/></e>
 <e lm="sagal" a="albertonl">   <i>sagal</i><par n="abric__n"/></e>
 <e lm="saleta" a="albertonl">   <i>salet</i><par n="abell/a__n"/></e>
-<e lm="semidéu" a="albertonl">   <i>semidéu</i><par n="abric__n"/></e>
+<e lm="semidéu" a="albertonl">   <i>semidé</i><par n="semid/éu__n"/></e>
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: Pay attention:  `<i>semid</i>`

File: apertium-cat.cat.dix [hunk 3]
```diff
@@ -5121,6 +5121,13 @@
 
 </pardef>
 
+<pardef n="semid/éu__n">
+  <e>       <p><l>éu</l>         <r>u<s n="n"/><s n="m"/><s n="sg"/></r></p></e>
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: remove "éu" on the right: `<r>éu<s n="n"/> ... ` (in the four entries).

Comment (albertonl) [reply apertium-cat.cat.dix]: What should I put instead of that?

File: apertium-cat.cat.dix [hunk 4]
```diff
@@ -5121,6 +5121,13 @@
...
+  <e>       <p><l>essa</l>      <r>u<s n="n"/><s n="f"/><s n="sg"/></r></p></e>
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: deessa: `<p><l>eessa</l>` and deesses...

Comment (albertonl) [reply apertium-cat.cat.dix]: Thanks.

File: apertium-cat.cat.dix (hunk 3 above)
Comment (jaumeortola) [reply apertium-cat.cat.dix]: Exactly this: 
` <e>       <p><l>éu</l>         <r>éu<s n="n"/><s n="m"/><s n="sg"/></r></p></e>`
The right part must be the same string in the paradigm name semid/**éu**__n

**Comment by jaumeortola** [comment]
Finally, this will be resolved differently with the other new diacritics. So I close the pull request.

**Pull request closed without merging**